tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -166,12 +166,6 @@ void TangoRosNode::onInit() {
   area_description_T_start_of_service_publisher_ =
       node_handle_.advertise<geometry_msgs::TransformStamped>(
           AREA_DESCRIPTION_T_START_OF_SERVICE_TOPIC_NAME, queue_size, latching);
-  point_cloud_publisher_ =
```

Comment:
What is the advantage of moving the publishers to the connect method? Did you measure noticeable performance gain by moving the publisher initialization to tango connect? One disadvantage is that publishers need to be reinitialized when doing a disconnect connect cycle.